scripts/build_imagemagick

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Here we make sure the script exits if any error occurs. All the steps are required and we don't want to build the binary with missing files or have to wait until the end of the script to see an unexpected error.

Reply:
TIL what `set -e` really means!